Deploy-CAM.ps1

```diff
@@ -202,6 +294,8 @@ public class TrustAllCertsPolicy : ICertificatePolicy {
                 username = $client
                 password = $key
                 tenantId = $tenant
+				ownerTenantId = $ownerTenant
+                ownerUpn = $ownerUpn
```

Comment:
I thought we already had an "email" field for registering users? Are we just not using it then?